cli/module_registry.go

```diff
@@ -699,6 +699,11 @@ func readModels(path string, logger logging.Logger) ([]ModuleComponent, error) {
 	defer vutils.UncheckedErrorFunc(func() error { return os.RemoveAll(parentAddr) })
 	parentAddr += "/parent.sock"
 
+	// in some scenarios, we want to run module code without having a "real"
+	// parent available. This environment variable illustrates this.
```

Comment:
let's move this generic description to `module/module.go`. this comment should explain why we run in "no module mode"

```suggestion
	// we start the module without connecting to a parent since we only need to perform validation and extract metadata
```